### PR TITLE
docs(ios): Help for 14.0 features

### LINKED ISSUE #4732: bug(ios): outdated documentation in app help
**Describe the bug**
I've noted the following issues with the iOS app's documentation that should be corrected as part of the 14.0 release:

iOS in-app help images:

"Finding a keyboard"
- [ ] Examples predate the keyboard search's implementation, and thus do not match the current state of the app.
- [ ] Does not note the local copy of the package we store for the user

"Installing custom keyboards"
- CTRL+F:  "All of the keyboards in the package are installed as a group."
    - [ ] No longer true.  Keyboards are installed based on the languages that are selected at install time.

"Sharing the File"
- [ ] Does not note that we literally give them a copy of EVERYTHING they install, not just what they ad-hoc install.
- [ ] Should have a bit at the top about "two ways":  QR code, local file.  QR code sharing feels kinda "hidden" without that.
- [ ] Typo in final paragraph: "Select[ing]," not "select."  The latter causes a verb tense conflict with the rest of the sentence..

### PR DESCRIPTION
**Pull request by jahorton**
Fixes #4732.

### CODE REVIEW

File: ios/help/index.md [hunk 1]
```diff
@@ -9,7 +9,8 @@ title: Keyman for iPhone and iPad 14.0 Help
 * [Version History](about/history)
 
 ## [Getting Started](start/)
-* [How To - Adding a Keyboard](start/installing-keyboards)
+* [How To - Adding a Keyboard](start/searching-for-keyboards)
+* [How To - Keyboard and Dictionary Installation](installing-packages)
```
Comment (darcywong00) [inline ios/help/index.md:13]:
```suggestion
* [How To - Keyboard and Dictionary Installation](start/installing-packages)
```

**Review by darcywong00 — APPROVED** [review]
lgtm

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 14.0.267-beta](https://keyman.com/downloads/releases/beta/14.0.267)

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 15.0.20-alpha](https://keyman.com/downloads/releases/alpha/15.0.20)

**Pull request merged**